tests/Backend/ExternalBuckets/BigqueryRegisterBucketTest.php

```diff
@@ -226,6 +226,88 @@ public function testRefreshBucketWhenSchemaDoesNotExist(): void
         $this->assertNotEmpty($bucket);
     }
 
+    /**
+     * @dataProvider provideComponentsClientTypeBasedOnSuite
+     */
+    public function testRegisterTableWithLongName(): void
+    {
+        $description = $this->generateDescriptionForTestObject();
+        $testBucketName = $this->getTestBucketName($description);
+        $bucketId = self::STAGE_IN . '.' . $testBucketName;
+
+        $this->dropBucketIfExists($this->_client, $bucketId, true);
+
+        $externalBucketBackend = 'bigquery';
+
+        $testClient = $this->_testClient;
+
+        $path = $this->prepareExternalBucketForRegistration($description);
+
+        $externalCredentials['connection']['backend'] = 'bigquery';
+        $externalCredentials['connection']['credentials'] = $this->getCredentialsArray();
+        $externalCredentials['connection']['schema'] = sha1($description) . '_external_bucket';
+
+        $db = WorkspaceBackendFactory::createWorkspaceBackend($externalCredentials);
+        // create table with long name, limit is 96 chars
+        $db->createTable(
```

Comment:
vyrobil bych tam jeste tabulky pro usecase , kdy to neprojde `-_A-Za-z0-9` . Na BQ jde vyrobit treba `moje nova 1234 -2ěščěš` a na snflk tam muzes mrsknout `$`. a stoji nas to tady jen vytvoreni tech tabulek